.github/workflows/social_media_message_scheduler/res/messages.yaml

```
@@ -0,0 +1,58 @@
+# the message name or id
+- submit_project_ideas:
+  #the list of target describes where you want your message to be sent it can be
+  # <twitter OR telegram_group OR telegram_channel >
+  targets:
+    - twitter
+  message: |
+    Hi all,
+
+    You have a project idea you think the Cameroonian community might benefit from ?
+
+    You can submit this idea to the osscameroon community and we can work on it together.
+
+    Click here to submit Down pointing backhand index
+    https://github.com/osscameroon/Open-Projects/issues/new?assignees=&labels=&template=project-idea-template.md&title=
+
+    Join us here https://t.me/joinchat/UpKZh_KXTaTx7JD7
+
+    #CaParleDev
+
+- bash_tutorial:
+  targets:
+    - twitter
+  message: |
```

Maybe we should consider working with multi-lang, something like :
```suggestion
  messages:
      -  message-fr: |
               ......
      -  message-en: |
               ......
```

What do you think ?

---

This is great, I will add it to the schema 👍